packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/ViewKeys/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, Icon, Text } from 'blockchain-info-components'
+import { FormattedMessage } from 'react-intl'
+import React from 'react'
+import styled from 'styled-components'
+
+const ContentWrapper = styled.div`
+  height: 19.5rem;
+  width: 19.5rem;
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem;
+  box-sizing: border-box;
+`
+
+const Header = styled(Text)`
+  margin-bottom: 1.5rem;
+`
+
+const Warning = styled(Text)`
+  margin-bottom: 0.25rem;
+`
+
+const Body = styled(Text)`
+  margin-bottom: 2rem;
+`
+
+const KeyIcon = styled(Icon)`
+  margin-bottom: 1.5rem;
+`
```

Comment:
We don't use rem anywhere else. We use `import media from services/ResponsiveService` and px